templates/course-results/lessons.php

```diff
@@ -144,7 +140,7 @@
 				}
 				?>
 
-				<h2>
+				<?php echo $course_has_lessons_in_modules ? '<h4>' : '<h3>'; ?>
```

Comment:
The `?>` on line 141 could be removed so that the PHP can continue without needing to add `<?php` again.